server/actions_apis.go

```diff
@@ -178,7 +179,11 @@ func (s *RegistryServer) ListApis(ctx context.Context, req *rpc.ListApisRequest)
 			if err != nil {
 				return nil, invalidArgumentError(err)
 			}
-			if !out.Value().(bool) {
+			v, ok := out.Value().(bool)
+			if !ok {
+				return nil, invalidArgumentError(errors.New("expression does not evaluate to a boolean"))
+			}
+			if !v {
 				continue
 			}
```

Comment:
Nit: Both of these if statements are related and can be combined as the only use of `v` and `ok`.
```
if v, ok := out.Value().(bool); !ok {
...
} else if !v {
...
}
```

Reply:
sgtm, updated